flex.css

```diff
@@ -0,0 +1,123 @@
+.app-header {
+  display: flex;
+  position: absolute;
+  padding: 10px 24px;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  background: #3c3a39;
+  z-index: 2;
+}
+
+.header {
+  display: flex;
+  flex: 4;
+  justify-content: flex-start;
+  position: absolute;
+  top: 75%;
+  left: 0;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.menu {
+  display: flex;
+  flex: 1;
+  justify-content: flex-end;
+  position: absolute;
+  top: 54%;
+  right: 0;
+  z-index: 1;
+}
+
+.app-homepage {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0;
+  top: 92px;
+  background: #1c1a19;
+}
+
+h1 {
+  width: 327px;
+  height: 124px;
+  top: 120px;
+  left: 24px;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+h2 {
+  position: static;
+  width: 372px;
+  height: 27px;
+  left: 24px;
+  top: 256px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  color: #fff5e1;
+}
+
+h3 {
+  position: static;
+  width: 327px;
+  height: 123px;
+  left: 24px;
+  top: 295px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+ul {
+  list-style: none;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+.github {
+  position: absolute;
+  width: 17.99px;
+  left: 0.08%;
+  right: 0.01%;
+}
+
+.link {
+  position: absolute;
+  left: 20%;
+  right: 75%;
+}
+
+.twitter {
+  position: absolute;
+  left: 40%;
+  right: 75%;
+}
+
+.background {
+  position: absolute;
+  left: 64%;
+  top: 80%;
+  width: 1139.56px;
+  height: 93.78px;
+  z-index: 3;
+}
```

Comment:
- [ ] The mobile view of your project is not fitting into mobile there is a blank space by the side and it is probably being caused by the `width: 375px;` given to `.app-head` class kindly rework your mobile head styling to fit
your design 👇 
        
![image](https://user-images.githubusercontent.com/66097371/148422996-b4b1d8e3-4227-472b-8071-0a6b340131f6.png)
figma design 👇 
       
![image](https://user-images.githubusercontent.com/66097371/148423206-7c808c30-8db2-4c46-956c-e8954662f342.png)